include/envoy/config/subscription.h

```diff
@@ -67,6 +67,11 @@ class SubscriptionCallbacks {
    * a RouteConfiguration, based on the underlying resource type.
    */
   virtual std::string resourceName(const ProtobufWkt::Any& resource) PURE;
+
+  /**
+   * Update cluster of management server.
```

Comment:
What does it means to update the cluster of the management server? Update it to what? There are no params..